planning/behavior_velocity_planner/include/utilization/util.hpp

```diff
@@ -137,6 +137,14 @@ double calcJudgeLineDistWithJerkLimit(
   const double velocity, const double acceleration, const double max_stop_acceleration,
   const double max_stop_jerk, const double delay_response_time);
 
+double calcMaxSlowDownVelocityFromDistanceToTarget(
+  const double max_slowdown_jerk, const double max_slowdown_accel, const double current_accel,
+  const double current_velocity, const double distance_to_target);
+
+double findReachTime(
+  const double j, const double a, const double v, const double d, const double min,
+  const double max);
```

Comment:
This is utility function. So, args of function should be more easy to read.
(jerk, accel, velocity...)

Reply:
I will change to jerk accel velocity

Reply:
[7a8d695](https://github.com/autowarefoundation/autoware.universe/pull/388/commits/7a8d69553edbacf057b51314b7f214777e5ce5ab)

Reply:
@yukkysaito 
it's min time and max time for finding target velocity
[ed5c9a6](https://github.com/autowarefoundation/autoware.universe/pull/388/commits/ed5c9a69992274b4cd80c96d1d6a4227fbe5a858)